src/QMCHamiltonians/PairCorrEstimator.cpp

```diff
@@ -28,17 +28,22 @@ PairCorrEstimator::PairCorrEstimator(ParticleSet& elns, std::string& sources)
   UpdateMode.set(COLLECTABLE, 1);
   num_species = elns.groups();
   n_vec.resize(num_species, 0);
-  for (int i(0); i < num_species; i++)
-    n_vec[i] = 1.0 / (elns.last(i) - elns.first(i));
+  for ( int i=0; i<num_species; i++ )
+    {
+      n_vec[i] = elns.last(i) - elns.first(i);
+    }
   N_e = elns.getTotalNum();
-  //use the simulation cell radius if any direction is periodic
-  if (elns.Lattice.SuperCellEnum)
+
+  // use the simulation cell radius if any direction is periodic
+  if ( elns.Lattice.SuperCellEnum )
   {
     Dmax   = elns.Lattice.WignerSeitzRadius;
     Volume = elns.Lattice.Volume;
   }
-  else
-    Volume = 1.0; //open bcs
+  else  // Open BC's
+    {
```

Comment:
Could you remove the curly brackets and save two lines?

Reply:
I initially submitted two PR's. One included only the unit test #2066 , while the other provided a fix to PairCorrEstimator to make the unit test pass #2067 . 

However, it seems my fork was behind mainline develop, so there were some issues with both. I updated my fork and re-implemented my changes in this new PR, which includes both the unit test and the fix. 

I've just pushed changes that address your comments above about brackets, as well as your additions in #2066 about the instantiation of PairCorrEstimator.

Apologies if I've bungled the process!